src/_utils/branding.ts

```diff
@@ -73,6 +73,12 @@ export const space: defaultBranding = {
   xxl: '48px',
 }
 
+export const horizontalSpace: defaultBranding = {
```

Comment:
maybe we should keep t-shirt naming convention `s`,  `m`, `l`,...

Reply:
I'd rather be explicit on their usage.